### ISSUE TITLE
Factor variables not rendered as integers

### ISSUE THREAD

**Issue by ndunnewind** [benjaminrich/table1#55]
Since v1.4, factor variable counts are rendered with decimal places:

![image](https://user-images.githubusercontent.com/43117768/120105460-45122800-c159-11eb-9a70-0dcfb03fd6e6.png)

An example in the vignette shows how to override this behavior by defining an alternative `render.categorical` function.

```r
my.render.cat <- function(x) {
    c("", sapply(stats.default(x), function(y) with(y,
        sprintf("%d (%0.0f %%)", FREQ, PCT))))
}
```

Shouldn't this be the default behavior? I think it makes sense to always show these counts as integers.

**Comment by benjaminrich** [comment]
You are quite right. Indeed, this should be the default (and I thought it was), but I must have introduced a bug somewhere. I have pushed a fix in commit a6a5f4b. Maybe you can test it out and let me know if it works for you?

**Comment by ndunnewind** [comment]
This seems to work! Thanks for the quick fix!
